react/__tests__/PlaceDetails.test.tsx

```diff
@@ -0,0 +1,61 @@
+import React from 'react'
+import { render } from '@vtex/test-tools/react'
+import {
+  mockSummaries as summaries,
+  completeAddress,
+  incompleteAddress,
+} from '../__mocks__/mockSummaries'
+import PlaceDetails from '../PlaceDetails'
+
+describe('Place Details', () => {
+  it('should render all data present in the summary and in the address', () => {
+    const { queryByText } = render(
+      <div>
+        <PlaceDetails address={completeAddress} summary={summaries[0]} />
```

Comment:
As far as I understood those `summaries` mocks are the same used on `CountryDescriptions` meaning they are `ARG`, `BRA` and `KOR` summaries.
To improve readability, you could give names to them in order to import as `summaries.ARG.summary`. It would help us to understand why one should render all data in address and summary versus only present in address.
```suggestion
        <PlaceDetails
          address={completeAddress}
          summary={summaries.ARG.summary}
        />
```
Of course you could name `mockDescriptions` instead of `mockSummaries` and use `descriptions.ARG.summary`.